src/backend/TrafficCourtsApi/citizen-api/Features/Lookups/LookupsService.cs

```diff
@@ -48,5 +54,10 @@ public Statute GetCountStatute(string countCode)
             if (_lookups == null) return null;
             return _lookups.statutes.FirstOrDefault(m=>m.code==Int32.Parse(countCode));
```

Comment:
This is error prone, is countCode an integer? is it null, is it empty?  Also use the type alias `int` instead of the structure name where possible